src/theory/arith/rewriter/node_utils.h

```diff
@@ -0,0 +1,163 @@
+/******************************************************************************
```

Comment:
Do any of these methods coincide with those in https://github.com/cvc5/cvc5/blob/master/src/theory/arith/arith_utilities.h?

I think I'd prefer modifying/extending that file instead of adding a new utility file.

Reply:
`arith_utilities` has `mkRationalNode` and `mkBoolNode` (which is called `mkConst` in this PR). Also it has `mkMult`, which is different from `mkMultTerm`, though. Given that the semantics of `mkMultTerm` are somewhat special (it assumes stuff about the monomial), I'd rather keep this separate/internal to the rewriter and not "advertise" it to the rest of arithmetic.